internal/test_cases/tokenize_test_case.go

```diff
@@ -16,9 +16,12 @@ import (
 // It is sent to the "tokenize" command of the executable,
 // the expected outputs are generated using the lox.ScanTokens function,
 // With that the output of the executable is matched.
+// Default value for PreserveWhitespaceWhileLogging is false.
+// We explicitly set it to true in the tests for whitespace.
 type TokenizeTestCase struct {
-	FileContents string
-	ExpectsError bool
+	FileContents                   string
+	ExpectsError                   bool
+	PreserveWhitespaceWhileLogging bool
```

Comment:
```suggestion
	HighlightWhitespaceWhileLogging bool
```

(More accurate representation of what we're doing compared to "preserve")